docs/sphinx/source/index.rst

```diff
@@ -34,6 +33,19 @@ application.
 There is a :ref:`variable naming convention <variables_style_rules>` to
 ensure consistency throughout the library.
 
+License
+=======
+
+`BSD 3-clause <https://github.com/pvlib/pvlib-python/LICENSE>`.
```

Comment:
```suggestion
`BSD 3-clause <https://github.com/pvlib/pvlib-python/LICENSE>`_.
```

I think that underscore will solve this:

![image](https://user-images.githubusercontent.com/57452607/128777928-f4cc6057-ce81-4098-baab-308c65519431.png)

Reply:
aha it's restructured text